docs/layout-of-source-files.rst

```diff
@@ -2,15 +2,23 @@
 Layout of a Solidity Source File
 ********************************
 
-Source files can contain an arbitrary number of contract definitions, include directives
-and pragma directives.
+Source files can contain an arbitrary number of
+:ref:`contract definitions<contract_structure>`, import_ directives
```

Comment:
Hmm, there's no section for imports yet?

Reply:
There is, in this file.

Reply:
Can you add a link to it then?

Reply:
Oh, quite cryptic.